concepts/eventing/performance evaluation/nats-jetstream/results_prod_HA.md

```diff
@@ -0,0 +1,240 @@
+# High Availibiltiy (HA) Evaluation of JetStream (Kyma: Production Profile)
+
```

Comment:
Can we have a short introduction sentence about what the document's purpose is ?

Reply:
added `## Purpose` section